examples/meta/src/evaluation/multiclass_ovr_evaluation.sg.in

```diff
@@ -0,0 +1,27 @@
+File f_feats_train = csv_file("@SHOGUN_DATA@/classifier_4class_2d_linear_features_train.dat")
+File f_labels_train = csv_file("@SHOGUN_DATA@/classifier_4class_2d_linear_labels_train.dat")
+
+#![create_features]
+Features feats_train = features(f_feats_train)
+Labels labels_true = labels(f_labels_train)
+#![create_features]
+
+#![create_classifier]
+Machine svm= machine("MulticlassLibLinear", C=1.0, labels=labels_true, seed=1)
```

Comment:
is the seed needed to pass the tests? can't imagine that

Reply:
Will remove it and see what happens